stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -48,6 +46,8 @@ spec:
               set -e
               set -x
 
+              chown -R 999 /data/db
```

Comment:
How about `chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /data/db` ?

Reply:
Yeah you're right, I didn't even considered this. I shouldn't be hard coding the values.

Reply:
Fixed!